cassdegrees/ui/views.py

```diff
@@ -273,15 +273,21 @@ def manage_courses(request):
 
         # to be implemented, currently has the sample model code
         elif action == 'Delete':
-            id_to_delete = post_data.get('id')
-
-            rest_api = requests.delete(model_api_url + id_to_delete + '/')
+            ids_to_delete= post_data.getlist('id')
```

Comment:
Maybe verify these are integers? Else buggy code could potentially do unwanted stuff here (they are passed directly into the url below)

Reply:
That shouldn't matter too much, the id's can't be altered by users (they are hidden input fields) and a post request with an unusual id made by something outside our Django app will be rejected since they don't have the required csrf token.